frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -278,6 +280,17 @@ function ReaderDictionary:addToMainMenu(menu_items)
                     end)
                 end,
             },
+            {
+                keep_menu_open = true,
+                text = _("Set dictionary priority for this book"),
+                help_text = _("This setting enables you to specify dictionary priorities on a per-book basis. Results from higher-priority dictionaries will be displayed first when looking up words. Only dictionaries that are currently active can be selected and prioritized."),
```

Comment:
Not really a "This setting" (like a checkbox would be), rather a dialog/feature.